src/extension.ts

```diff
@@ -212,7 +212,7 @@ async function _activateExtension(
   const schemasViewProvider = SchemasViewProvider.getInstance();
   const statementsViewProvider = FlinkStatementsViewProvider.getInstance();
   const flinkDatabaseViewProvider = FlinkDatabaseViewProvider.getInstance();
-  const supportViewProvider = new SupportViewProvider();
+  const helpCenterViewProvider = new HelpCenterViewProvider();
```

Comment:
thought renaming view provider would help with codebase findability, even though it's not strictly necessary